deployment/terraform/variables.tf

```diff
@@ -0,0 +1,157 @@
+variable "project" {
+  default = "OpenApparelRegistry"
```

Comment:
We typically put spaces between these since they're expected to be tag values vs. names.